pages/_en-US/ds-index/hardmod.md

```diff
@@ -11,4 +11,90 @@ Hardmodding is when you physically solder the motherboard of the Nintendo DSi to
+
+#### Soldered microSD adapter example
+![microSD example](/assets/images/dsi-harmod/sd.jpg)
+
+- Insert the microSD card adapter to your computer
+  - **WARNING** - Windows will ask you to format it: **DO NOT** format it - this might cause irrepairable damage
+
+#### Windows instructions (skip for macOS / Linux)
+
+1. Open Win32DiskImager
+2. Click the folder icon and browse to your desktop. In the text box, write NAND_0.bin. When selecting type, choose All types *.* 
+3. Choose the device that is the DSi and click read. 
+4. Once it is done, click the folder icon,change the name to NAND_1.bin and read again
+5. Open HxD and drag both files into the editor. Go to the top bar, click "Analysis",click "File compare" from the drop down menu, then click "Compare".
+6. Choose to compare both files then click OK when done
+  - If it says "The chosen files are identical.", you are good to go to the next section
+  - If it doesn't say that, and both NANDs aren't around 240MB, dump NAND_1/NAND_0 again
+7. Open Win32DiskImager, click the folder icon and choose your existing NAND backup you created before.
+8. Flash it with the `Write` button.
+9. Unplug the SD card adapter and attempt to turn on the DSi
+
+#### Linux / macOS instructions (skip if on Windows)
+
+
+1. Work out where the SD card is mounted
+  - Linux:
+    1. Unplug the SD card adapter
+    2. Run `lbslk` on a terminal
+    3. Plug in the SD card adapater
+    4. Run `lbslk` again
+    5. This time a new device should appear
+    6. It may be called `/dev/sdb`, make sure you note down what it is called on *your* terminal
+
+  - macOS:
+    1. Unplug the SD card adapter
+    2. Run `df` on a terminal
+    3. Plug in the SD card adapter
+    4. Run `df` again
+    5. A new device should appear in the list
+    6. It may be called `/dev/disk1s1`, make sure you note down what it is called on *your* terminal/
+
+2. Dump the NAND
+  - Run the following command on a terminal:
+    - `cat {device-name} > nand0.bin` 
+    - Replace {device-name} with the SD card mount location
+    - E.g `cat /dev/sdb > nand0.bin`
+    - Then run `cat {device-name} > nand1.bin`
+
+3. Compare NAND dumps
+  1. Run the following command:
+    - Linux : `md5sum nand0.bin nand1.bin`
+    - macOS : `md5 -r nand0.bin nand1.bin`
+  2. Check that the hash generated matches each other.
+  3. If not, redump the NAND, adjusting the wiring if necessary
+  4. Keep on redumping the NAND until the hashes are identical
+
+4. Flash NAND
+  1. Locate the NAND backup you created before on the console.
+  2. Run the following command:
+    - `cat {existing-nand-backup} > {device-name}`
+    - Replace `{existing-nand-backup}` with the location and name of your old NAND backup
+    - Replace `{device-name}` with the mount point of the SD card
+  3. At this point you may unplug the SD card adapter and attempt to turn on the DSi.
+  4. If all went well, the DSi should have booted to the state it was when the backup was created!
```

Comment:
This isn't really a step. You should put this and the former one in a new line

Reply:
fyi to highlight two lines you can click and drag the blue [+] thing

Reply:
Interesting to note. Thank you